install-ansible.sh

```diff
@@ -25,15 +39,10 @@ elif [[ "Ubuntu" =~ $os_VENDOR || "LinuxMint" =~ $os_VENDOR ]]; then
   apt-get update
   apt-get install -y ansible
 elif [[ "RedHatEnterpriseServer" =~ $os_VENDOR || "CentOS" =~ $os_VENDOR || -r /etc/redhat-release ]]; then
```

Comment:
I think we can remove  ` -r /etc/redhat-release` as we will never enter this condition.